runtime/interface.go

```diff
@@ -98,6 +98,8 @@ type ACLs interface {
 }
 
 type Tables interface {
+	// SetTable create or update a stick-table entry in the table.
+	SetTable(table, key string, dataType models.StickTableEntry, process int) error
```

Comment:
Can we rename this to SetTableEntry so it better represents what this method does?